crates/shared/src/arguments.rs

```diff
@@ -165,3 +175,26 @@ pub fn wei_from_gwei(s: &str) -> anyhow::Result<f64> {
     let in_gwei: f64 = s.parse()?;
     Ok(in_gwei * 1e9)
 }
+
+impl FromStr for RateLimitingStrategy {
+    type Err = anyhow::Error;
+
+    fn from_str(config: &str) -> Result<Self> {
+        let mut parts = config.split(',');
+        let back_off_growth_factor = parts
+            .next()
+            .ok_or_else(|| anyhow::anyhow!("missing back_off_growth_factor"))?;
+        let min_back_off = parts
+            .next()
+            .ok_or_else(|| anyhow::anyhow!("missing min_back_off"))?;
+        let max_back_off = parts
+            .next()
+            .ok_or_else(|| anyhow::anyhow!("missing max_back_off"))?;
```

Comment:
ubernit: Should we also check there are no extra parts?
```suggestion
        ensure!(parts.next().is_none(), "extraneous rate limiting parameters");
```